src/scout_apm/core/tracked_request.py

```diff
@@ -111,7 +116,15 @@ def tag(self, key, value):
     # In seconds
     def duration(self):
         if self.end_time is not None:
-            (self.end_time - self.start_time).total_seconds()
+            return (self.end_time - self.start_time).total_seconds()
         else:
             # Current, running duration
-            (datetime.utcnow() - self.start_time).total_seconds()
+            return (datetime.utcnow() - self.start_time).total_seconds()
+
+    # Add any interesting annotations to the span. Assumes that we are in the
+    # process of stopping this span.
+    def annotate(self):
+        slow_threshold = 0.0
```

Comment:
Our [Ruby logic for this](https://github.com/scoutapp/scout_apm_ruby/blob/c805e10520231390eaaadb8aad6258ba0aad6ca9/lib/scout_apm/tracked_request.rb#L146)...we set the threshold at 500 ms.

Reply:
Yep - set to 0 for testing purposes for now. 500ms or so is reasonable as a default.